README.md

```diff
@@ -8,14 +8,15 @@
 
 ## About
 This library simplifies integration of Sentry's
-[node-raven](https://docs.sentry.io/clients/node/) library with AWS Lambda.
+[@sentry/node](https://docs.sentry.io/error-reporting/quickstart/?platform=node) library with AWS Lambda.
 The only supported platforms are Node.js 6.10 and 8.10. Python and Java
 support will require dedicated libraries. Pull requests are welcome!
 
 ### What is Raven and Sentry?
-It's a bit confusing, but _Raven_ is the official name of the error reporting SDK
-that will forward errors, exceptions and messages to the _Sentry_ server. For
-more details of what Raven and Sentry actually is, refer to the official Sentry documentation: https://docs.sentry.io/.
+It's a bit confusing, but _Raven_ *was* the old official name of the error reporting SDK
+that will forward errors, exceptions and messages to the _Sentry_ server. The new name is
+_Unified Node.js SDK_. 
+For more details of what Raven and Sentry actually is, refer to the official Sentry documentation: https://docs.sentry.io/.
```

Comment:
I'd maybe add something stating the old Raven SDK is still available using v1.1.2? Maybe some people would still look for the old implementation.

Reply:
updated the Readme to point to old Version